src/go/app/vrouter.go

```diff
@@ -55,19 +56,18 @@ type Emulator struct {
 	Egress     []string `mapstructure:"egress"`
 	Name       string   `mapstructure:"name"`
 	Bandwidth  string   `mapstructure:"bandwidth"`
-	Burst      string   `mapstructure:"burst"`
```

Comment:
Why was burst removed?

Reply:
Because it's in [the official docs](https://docs.vyos.io/en/latest/configuration/trafficpolicy/index.html#cfgcmd-set-qos-policy-network-emulator-policy-name-burst-burst-size), but when you try to tab-complete it inside the running image, there's no `burst` option ¯\_(ツ)_/¯